apps/studio/components/interfaces/Settings/Logs/LogSelection.tsx

```
@@ -70,6 +70,19 @@ const LogSelection = ({ log, onClose, queryType, isLoading, error }: LogSelectio
         return <DefaultPreviewSelectionRenderer log={apiLog} />
 
+      case 'multigres':
+        const parsedMultigresMessage = (() => {
+          if (typeof log.event_message !== 'string') return null
+          try {
+            const parsed = JSON.parse(log.event_message)
+            return parsed && typeof parsed === 'object' && !Array.isArray(parsed) ? parsed : null
+          } catch {
+            return null
+          }
+        })()
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)